<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VectorAnalysisVisualWidget</class>
 <widget class="QWidget" name="VectorAnalysisVisualWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QSplitter" name="splVertical">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frDiags">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QSplitter" name="splHorizontal">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="AccumulationFunction" name="wgtAccumulationFunc" native="true"/>
         <widget class="VectorCloud" name="wgtVectorCloud" native="true"/>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frCloudToolbar">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnPlus">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../kernel.qrc">
              <normaloff>:/images/Plus.png</normaloff>:/images/Plus.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMinus">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../kernel.qrc">
              <normaloff>:/images/Minus.png</normaloff>:/images/Minus.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTreeView" name="tvFactors"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccumulationFunction</class>
   <extends>QWidget</extends>
   <header location="global">accumulationfunction.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VectorCloud</class>
   <extends>QWidget</extends>
   <header location="global">vectorcloud.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../kernel.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>splVertical</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>VectorAnalysisVisualWidget</receiver>
   <slot>splitterMoved(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>724</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splHorizontal</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>VectorAnalysisVisualWidget</receiver>
   <slot>splitterMoved(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>725</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlus</sender>
   <signal>clicked()</signal>
   <receiver>VectorAnalysisVisualWidget</receiver>
   <slot>on_plusButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMinus</sender>
   <signal>clicked()</signal>
   <receiver>VectorAnalysisVisualWidget</receiver>
   <slot>on_minusButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>splitterMoved(int,int)</slot>
  <slot>on_plusButtonClicked()</slot>
  <slot>on_minusButtonClicked()</slot>
 </slots>
</ui>
